app/packs/src/apps/admin/ThirdPartyApp.js

```diff
@@ -266,7 +266,7 @@ export default class ThirdPartyApp extends React.Component {
         <Modal.Body style={{ overflow: 'auto' }}>
           <div className="col-md-12">
             <FormGroup controlId="formControlName">
-              <Col componentClass={ControlLabel} sm={3}>
+              <Col as={ControlLabel} sm={3}>
```

Comment:
ControlLabel is a legacy Component and should be replaced with Form.Label

Reply:
I only wanted to get rid of the warning regarding the unknown `componentClass` prop. @ashwiniHerle has marked this and the `UserLabels` component as pending so I did not want to change too much.